api/v1alpha1/osbuildenvconfig_webhook.go

```diff
@@ -91,6 +108,17 @@ func (r *OSBuildEnvConfig) ValidateCreate() error {
 	return nil
 }
 
+func validateUniqueWorkerNames(workers []WorkerConfig) error {
+	workerNames := make(map[string]bool)
```

Comment:
Not that is a big deal but `map[string]struct{}` has better memory utilization (`struct{}{}` doesn't take any memory),

Reply:
Sure. What do I set when adding a key to the map? `workerNames[worker.Name] = ???`

Reply:
Done